.gitpod.Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM gitpod/workspace-full
 SHELL ["/bin/bash", "-c"]
 
-# Install projector and PhpStorm
+# Install projector 
 RUN sudo apt-get -qq update && sudo apt-get -qq install -y python3 python3-pip libxext6 libxrender1 libxtst6 libfreetype6 libxi6
 RUN pip3 install projector-installer
-# Fragile - will break when they change options
+# Install PhpStorm
 RUN mkdir -p ~/.projector/configs  # Prevents projector install from asking for the license acceptance
-RUN printf "8\nY\n3\n" | projector install --no-auto-run
+RUN projector install 'PhpStorm 2020.3.3'
```

Comment:
Were you able to do this manually? I'd still expect you to need the -`-no-auto-run`

Reply:
Thank you, that explains why my prebuild froze